scrapy_webarchive/downloadermiddlewares.py

```diff
@@ -56,11 +56,10 @@ def process_request(self, request: Request, spider: Spider):
         if not warc_record:
             self.stats.inc_value("webarchive/response_not_found", spider=spider)
             return Response(url=request.url, status=404)
-        
-        request.meta["warc_record_response"] = warc_record
-        
+
         # Record found, try to re-create a response from it.
         response = record_transformer.response_for_record(warc_record, request)
+        response.meta["response_origin"] = warc_record
```

Comment:
I think that actually all `meta` keys need to start with `webarchive_`, to avoid naming conflicts.
Origin sounds like HTTP Origin, maybe it's better to use something with warc here?

Reply:
See https://github.com/q-m/scrapy-webarchive/pull/31#issuecomment-2615882022